pkgs/games/space-station-14-launcher/space-station-14-launcher.nix

```diff
@@ -0,0 +1,143 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+, wrapGAppsHook
+, iconConvTools
+, copyDesktopItems
+, makeDesktopItem
+, libX11
+, libICE
+, libSM
+, libXi
+, libXcursor
+, libXext
+, libXrandr
+, fontconfig
+, glew
+, SDL2
+, glfw
+, glibc
+, libGL
+, freetype
+, openal
+, fluidsynth
+, gtk3
+, pango
+, atk
+, cairo
+, zlib
+, glib
+, gdk-pixbuf
+}:
+let
+  version = "0.20.5";
+  pname = "space-station-14-launcher";
+in
+buildDotnetModule rec {
+  inherit pname;
```

Comment:
```suggestion
  inherit pname version;
```

Reply:
Doing this would break the launcher, as `buildDotnetModule` will use this to version the built SS14.Loader assembly (which shouldn't be versioned) and then the downloaded engine assemblies won't be able to load it properly.